components/NewPersonView/PersonTimeline.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { formatDistanceToNow } from 'date-fns';
+import { Case } from 'types';
+import useSearch from 'hooks/useSearch';
+import s from './index.module.scss';
+import FilterButton, { Filter } from './FilterButton';
+import { Submission } from 'data/flexibleForms/forms.types';
+import SearchBox from 'components/SubmissionsTable/SearchBox';
+import UnfinishedSubmissionsEvent from './UnfinishedSubmissions';
+import { normaliseDateToISO } from 'utils/date';
+import Event from './Event';
+import { mockSubmission } from 'factories/submissions';
+
+const minorForms = ['ASC_case_note', 'Child Case Note'];
```

Comment:
What do we decide is minor vs major?

It would be more sensible to store this information in the root level data folder than in our component code.

Reply:
yes, we should do that, but at the moment i don't have a good sense of what's major and minor - it will probably be a longer list than just these two.